tools/gn

```diff
@@ -222,6 +226,9 @@ def to_gn_args(args):
     elif args.goma and os.path.exists(depot_tools_bin_dir):
       gn_args['use_goma'] = True
       gn_args['goma_dir'] = depot_tools_bin_dir
+    elif args.goma:
+      print("GOMA usage was specified but can't be found, set the GOMA_DIR environment variable.")
+      sys.exit(1)
```

Comment:
I'd just leave it with an error and not exit. Since --goma defaults to true which is still reasonably convenient. Otherwise most of our non-google users will run gn --some other flags, fail, then re-run gn --some other flags --no-goma, which adds some friction.

Reply:
done